ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/sequence/ActionNodeState.java

```diff
@@ -58,6 +58,49 @@ public ActionNodeState(long id, D definition, CRDTInfo crdtInfo)
       orientationDistanceToGoalTolerance = new CRDTStatusDouble(ROS2ActorDesignation.ROBOT, crdtInfo, Double.NaN);
    }
 
+   /**
+    * Updates the definition executeAfterActionName string for
+    * saving an up to date human readable name in the JSON.
+    * It also finds the correct node upon loading the name from JSON.
+    */
+   public void updateAndValidateExecuteAfter(List<ActionNodeState<?>> actionStateChildren)
```

Comment:
Moved this function from the RDX, which needs to be called as well in the executor. Not possible before all fields were bidirectional.